<!DOCTYPE HTML>
<!--
	Strata by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>publications</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="../assets/css/main.css" />
		<style>
			.comment {
				border: 1px solid #ccc; /* Creates a border around the comment */
				margin: 10px 0; /* Adds some space above and below the comment */
				padding: 10px; /* Adds some space inside the comment box */
				background-color: #f9f9f9; /* Light background for the comment */
				font-size: 0.9em; /* Adjust the font size to differentiate from the main list */
			}
		</style>
	</head>
	<body class="is-preload">

		<!-- Header -->
			<header id="header">
				<div class="inner">
					<a href="../index.html" class="image avatar"><img src="../images/profiles/2.png" alt=""  style="transform: scale(2);" /></a>
					<br />
					<br />
					<br />
					<a href="../index.html"><h1 style="margin-top: 10px;"><strong>Home</strong></h1></a>
					<a href="../htmls/research_topics.html"><h1 style="margin-top: 10px;"><strong>Research</strong></h1></a>
					<a href="../htmls/publications.html"><h1 style="margin-top: 10px;"><strong>Publications</strong></h1></a>
					<a href="../htmls/news.html"><h1 style="margin-top: 10px;"><strong>News</strong></h1></a>
					<a href="../htmls/photos.html"><h1 style="margin-top: 10px;"><strong>Photos</strong></h1></a>
				</div>
			</header>

		<!-- Main -->
			<div id="main">

				<!-- One -->
					<section id="one">
						<head>
    <title>Publications</title>
</head>
<body>
    <h2>Peer-reviewed, first authorship</h2>
    <ol>
        <li>Lingjie Meng, Tom O. Delmont, Morgan Gaïa, Eric Pelletier, Antonio Fernàndez-Guerra, Samuel Chaffron, Junyi Wu, Hiroto Kaneko, Hisashi Endo, Hiroyuki Ogata. (2023). Genomic adaptation of giant viruses in polar oceans. <em>Nature Communications</em>, 14: 6233.
			<div class="comment">
                Giant viruses, despite not being defined as 'life', possess large number of genes. Previous research has confirmed that marine giant viruses tend to be endemic, which inspires me to work on the environmental adaptation of giant viruses. In this study, it is suggested that giant viruses have adapted to polar environments through changes in their gene repertoire, and their adaptation mechanism may differ from others. However, current research remains at the hypothetical stage, and especially since there are many unexplained genes in viruses, a definitive virus adaptation mechanism has not been identified.
            </div>
        <li>Morgan Gaïa †, Lingjie Meng †, Eric Pelletier, Patrick Forterre, Chiara Vanni, Antonio Fernandez-Guerra, Olivier Jaillon, Patrick Wincker, Hiroyuki Ogata, Mart Krupovic, Tom O. Delmont. (2023). Mirusviruses link herpesviruses to giant viruses. <em>Nature</em>, 616(7958), 783–789.</li>
		<div class="comment">
			Most of double stranded DNA viruses are classified into one of two realms: Varidnaviria, which includes giant viruses, and Duplodnaviria, which includes herpesviruses causing diseases in animals. These two realms have been considered evolutionarily unrelated. However, the newly discovered mirusviruses possess features of both realms. Genes of “informational module” for genome replication and gene expression were similar to those of Varidnaviria, while genes of “virion module” that encode structural proteins were similar to those of Duplodnavira. This mosaic feature of mirusviruses suggests that herpesviruses evolved from tailed bacterial viruses via ancestral protist-infecting viruses, namely the ancestors of the modern day mirusviruses and herpesviruses. Furthermore, gene transfers of “informational genes” between the two realms probably played crucial roles in their evolution. Mirusviruses appear to be prevalent and abundant in the sea and likely infect a variety of protists. However, they were discovered by bioinformatics analyses of marine metagenomic data, and there are currently no cultured viruses from this group of viruses.
		</div>
		<li>Lingjie Meng, Hisashi Endo, Romain Blanc-Mathieu, Samuel Chaffron, Rodrigo Hernández-Velázquez, Hiroto Kaneko, Hiroyuki Ogata. (2021). Quantitative assessment of nucleocytoplasmic large DNA virus and host interactions predicted by co-occurrence analyses. <em>mSphere</em>, 6(2):e01298-20</li>
        <div class="comment">
			Most of double stranded DNA viruses are classified into one of two realms: Varidnaviria, which includes giant viruses, and Duplodnaviria, which includes herpesviruses causing diseases in animals. These two realms have been considered evolutionarily unrelated. However, the newly discovered mirusviruses possess features of both realms. Genes of “informational module” for genome replication and gene expression were similar to those of Varidnaviria, while genes of “virion module” that encode structural proteins were similar to those of Duplodnavira. This mosaic feature of mirusviruses suggests that herpesviruses evolved from tailed bacterial viruses via ancestral protist-infecting viruses, namely the ancestors of the modern day mirusviruses and herpesviruses. Furthermore, gene transfers of “informational genes” between the two realms probably played crucial roles in their evolution. Mirusviruses appear to be prevalent and abundant in the sea and likely infect a variety of protists. However, they were discovered by bioinformatics analyses of marine metagenomic data, and there are currently no cultured viruses from this group of viruses.
		</div>
		<li>Lingjie Meng, Yishuai Du, Pengfei Liu, Xian Li, Ying Liu. (2017). Involvement of LuxS in Aeromonas salmonicida metabolism, virulence and infection in Atlantic salmon (Salmo salar L). <em>Fish & shellfish immunology</em>, 64, 260–269.</li>
		<div class="comment">
			Quorum sensing is a communication system used by bacteria that depends on their population density and involves signaling molecules called autoinducers (AI). Autoinducer-2 (AI-2), produced by the LuxS enzyme, is known to facilitate interspecies communication. In Aeromonas salmonicida, a pathogen affecting cold-water fish, the role of AI-2 and LuxS is not well understood. This study aimed to explore LuxS's function in A. salmonicida. The findings revealed that wild-type A. salmonicida produces AI-2, while a luxS mutant does not. LuxS deficiency was linked to changes in bacterial morphology, surface properties, and virulence. Additionally, when salmon were infected with the luxS mutant, immune responses were triggered earlier. The study hypothesizes that AI-2 quorum sensing may regulate the expression of the A-layer protein gene vapA, influencing bacterial survival under immune attack.
		</div>
	</ol>
    <p>† Co-first author</p>
    
    <h2>Peer-reviewed, contributing authorship</h2>
    <ol>
		<li>Liwen Zhang, Lingjie Meng, Yue Fang, Hiroyuki Ogata, and Yusuke Okazaki. (2024). Spatiotemporal dynamics revealed the dark water community of giant virus from a deep freshwater lake. <em>The ISME Journal</em>.</li>
		<li>Junyi Wu, Lingjie Meng, Morgan Gaïa, Hiroyuki Hikida, Yusuke Okazaki, Hisashi Endo, Hiroyuki Ogata. (2024). Gene transfer among viruses substantially contributes to gene gain of giant viruses. <em>Molecular Biology and Evolution</em>.</li>
		<li>Hongda Zhao, Lingjie Meng, Hiroyuki Hikida, and Hiroyuki Ogata. (2024). Eukaryotic genomic data uncover an extensive host range of mirusviruses. <em>Current Biology</em>.</li>
        <li>Hongda Zhao, Ruixuan Zhang, Junyi Wu, Lingjie Meng, Yusuke Okazaki, Hiroyuki Hikida, Hiroyuki Ogata. (2023). A 1.5 Mb continuous endogenous viral region in the arbuscular mycorrhizal fungus Rhizophagus irregularis. <em>Virus Evolution</em>.</li>
        <li>Pengfei Liu, Yishuai Du, Lingjie Meng, Xian Li, Dong Yang, Ying Liu. (2019). Phosphoproteomic analyses of kidneys of Atlantic salmon infected with Aeromonas salmonicida. <em>Scientific reports</em>, 9(1):2101.</li>
        <li>Yishuai Du, Pengfei Liu, Lingjie Meng, Zaki Sharawy, Ying Liu. (2018). Colonization of Aeromonas salmonicida subsp. masoucida strains in Atlantic salmon (Salmo salar L.) during infection. <em>Aquaculture Research</em>, doi.org/10.1111/are.13637.</li>
        <li>Peng-fei Liu, Yishuai Du, Lingjie Meng, Xian Li, Ying Liu. (2017). ALDH7A1 is a protein that protects Atlantic salmon against Aeromonas salmonicida at the early stages of infection. <em>Fish & shellfish immunology</em>, 70:30-39.</li>
        <li>Yishuai Du, Ying Liu, Peng Xiao, Lingjie Meng, Pengfei Liu. 2017. Development and Application of a Quantitative Real‐time Polymerase Chain Reaction Assay for the Detection of Aeromonas salmonicida. <em>Journal of the World Aquaculture Society</em>, https://doi.org/10.1111/jwas.12395.</li>
        <li>Pengfei Liu, Yishuai Du, Lingjie Meng, Xian Li, Ying Liu. (2017). Proteomic analysis in kidneys of Atlantic salmon infected with Aeromonas salmonicida by iTRAQ. <em>Developmental & Comparative Immunolog</em>, 72:140-153.</li>
        <li>PengFei Liu, Yishuai Du, Lingjie Meng, Xian Li, Ying Liu. (2016). Metabolic profiling in kidneys of Atlantic salmon infected with Aeromonas salmonicida based on H-1 NMR. <em>Fish & Shellfish Immunology</em>, 58:292-301.</li>
        <li>Xian Li, Liang Chi, Huiqin Tian, Lingjie Meng, Jimeng Zheng, Xiaolong Gao, Ying Liu. (2016). Colour preferences of juvenile turbot (Scophthalmus maximus). <em>Physiology & Behavior</em>, 156:64-70.</li>
        <li>Yishuai Du, Mengmeng Yi, Peng Xiao, Lingjie Meng, Xian Li, Guoxiang Sun, Ying Liu. (2015). The impact of Aeromonas salmonicida infection on innate immune parameters of Atlantic salmon (Salmo salar L). <em>Fish & Shellfish Immunology</em>, 44(1):307-15.</li>
    </ol>
</body>
					</section>

		<!-- Footer -->
		<footer id="footer">
			<div class="inner">
				<ul class="icons">
					<li><a href="https://twitter.com/lingjie_marine" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
					<li><a href="https://github.com/LingjieEcoEvo" class="icon brands fa-github"><span class="label">Github</span></a></li>
					<li><a href="mailto:lingjie@kuicr.kyoto-u.ac.jp" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
				</ul>
				<ul class="copyright">
					<li>&copy; v202404</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
				</ul>
			</div>
		</footer>

		<!-- Scripts -->
			<script src="../assets/js/jquery.min.js"></script>
			<script src="../assets/js/jquery.poptrox.min.js"></script>
			<script src="../assets/js/browser.min.js"></script>
			<script src="../assets/js/breakpoints.min.js"></script>
			<script src="../assets/js/util.js"></script>
			<script src="../assets/js/main.js"></script>

	</body>
</html>