<!DOCTYPE HTML>
<!--
	Strata by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Research</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="../assets/css/main.css" />
	</head>
	<body class="is-preload">

		<!-- Header -->
		<header id="header">
				<div class="inner">
					<a href="../index.html" class="image avatar"><img src="../images/profiles/2.png" alt=""  style="transform: scale(2);" /></a>
					<br />
					<br />
					<br />
					<a href="../index.html"><h1 style="margin-top: 10px;"><strong>Home</strong></h1></a>
					<a href="../htmls/research_topics.html"><h1 style="margin-top: 10px;"><strong>Research</strong></h1></a>
					<a href="../htmls/publications.html"><h1 style="margin-top: 10px;"><strong>Publications</strong></h1></a>
					<a href="../htmls/news.html"><h1 style="margin-top: 10px;"><strong>News</strong></h1></a>
					<a href="../htmls/photos.html"><h1 style="margin-top: 10px;"><strong>Photos</strong></h1></a>
				</div>
			</header>

		<!-- Main -->
			<div id="main">

				<!-- Some selected research topics -->
				<header class="major" style="text-align: left; margin-top: 20px; font-size: 14pt; line-height: 1.6; color: #000;">
					<span style="font-weight: 1000;font-size: 36pt;color: #052659;">Research Topics</span>
					<p style="font-style: italic; color: #555;"> Ecology, Evolution, and Bioinformatics</p>
				</header>
				<section id="research-topics" style="text-align: center; margin-bottom: 30px;color: #000;">
					<a href="#one" class="button">Diversity & Evolution</a>
					<a href="#two" class="button">Host & Ecology</a>
					<a href="#three" class="button">Virus-Host interaction</a>
					<a href="#three" class="button">Viral functions</a>
				</section>
				
				<section id="research-topics" style="text-align: center; margin-bottom: 30px;color: #000;">
					
				<header class="major" style="text-align: left; margin-top: 20px; font-size: 14pt; line-height: 1.6; color: #000;">
					<span style="font-weight: 1000; font-size: 18pt;">Introduction</span>
					<br /><br />
					<p style="text-indent: 2em; margin-bottom: 15px;color: #000;">
						I have been studying giant viruses since I began my Ph.D. program at Kyoto University in 2019. At that time, there was no universal approach to identify this group of eukaryotic viruses with giant genomes. Shortly afterward, two major papers were published in 2020, using metagenomics to study giant viruses. At the same time, the ICTV officially classified this group of viruses as Nucleocytoviricota.
						These developments were inspiring and significantly influenced the direction of my research. During my Ph.D., I have been using metagenomics to investigate various aspects of marine giant viruses.
					</p>
					<p style="text-indent: 2em; margin-bottom: 15px;color: #000;">
						Metagenomics is essential for studying giant viruses because it enables the discovery and analysis of these viruses directly from environmental samples in a much larger broad. Whereas traditional approach is limited to approximately 100 species-level isolates obtained through laboratory co-cultivation with a narrow range of protists and algae.
						Metagenomics reveals the vast diversity and global distribution of giant viruses, many of which possess complex genomes with genes related to host manipulation and metabolism. Metagenomics helps answer key questions about their evolutionary relationships, functional potential, and ecological roles, such as how they influence microbial communities and biogeochemical cycles.
					</p>
					<p style="text-indent: 2em; margin-bottom: 15px;color: #000;">
						However, each approach has its limitations. For example, it is painful for me to discover thousands of giant viruses without knowing which hosts they infect. I can only generate hypotheses without being able to validate them. While I enjoy my work, I am not satisfied at the same time.  I give me a small, fundamental but general questions, and for each one, I am deeply eager to find the answers:
						Who are they (Biodiversity)? Where are they (Biogeography)? When do they appear (Dynamics)? What do they infect (Host range)? How do they infect (Infection cycle)? Why do they become so (Evolution)?
					</p>
					<p style="text-indent: 2em;color: #000;">
						These questions are simple yet complex. While some have been partially answered, this is not the end. I aspire to push the boundaries of knowledge further through my efforts. To achieve this, I have structured my work around few directions.
					</p>
				</header>
				</section>

				<!-- One -->
				<section id="one" style="display: flex; flex-wrap: wrap;">
					<!-- Image Section -->
					<div class="profile-photo" style="flex-basis: 30%; display: flex; justify-content: center; align-items: center; flex-direction: column;">
						<img src="../images/research/virus_diveristy.png" alt="Lingjie Meng" style="max-width: 100%; height: auto; border-radius: 10px;" />
						<header class="major" style="text-align: center; margin-top: 10px; font-size: 8pt;">
							<h1><span style="font-weight: 1000;">Protist dsDNA virus (ICTV)<br />
						</header>
					</div>

					<!-- Profile Information Section -->
					<div class="profile-info" style="flex-basis: 70%; padding: 20px; font-size: 14pt;">
						<span style="font-weight: 1000; font-size: 18pt;">Biodiversity & Evolution</span>
						<br /><br />
						<p style="text-indent: 2em; color: #000;">Currently, the ICTV database includes around 10,000 experimentally validated virus species. However, metagenomic analyses have revealed orders of magnitude more viruses, many of which were previously unculturable and remained undetected by traditional methods. </p>
						<p style="text-indent: 2em; color: #000;">Generally, detecting viral diveristy is challenging, because but not limited to the following reasons:</p>
						<ul>
							<li>Giant viruses are generally less abundant than cellular organisms in environmental samples and have smaller genomes</li>
							<li>Viruses exhibit high diversity but lack a universal marker gene like cellular organisms</li>
							<li>Giant viral genomes often contain many repetitive sequences and high intraspecies diversity, making it difficult to obtain complete genome assemblies</li>
							<li>There are only few reference genomes with isolated hosts</li><br/>
							<p style="text-indent: 2em; color: #000;"> In bioinformatics, what we can do is improve the pipeline and our logic of finding viruses. A particularly effective approach is to focus on the most informative genes. By doing so, we discovered a phylum-level lineage related to herpesviruses, named "Mirusviricota" <a href="https://www.nature.com/articles/s41586-023-05962-4">(Nature 2023)</a>.
								Another approach is to define a flexible core gene set criterion for giant viruses, which helps identify a large group of FSV-related giant viral genomes <a href="https://journals.asm.org/doi/10.1128/msystems.01168-24">(mSystems 2024)</a>.
								In addition to free-living virions, screening for viral regions in eukaryotic genomes has also surprisingly revealed a large amount of viral genomic information <a href="https://www.biorxiv.org/content/10.1101/2025.04.19.649669v2">(preprint 2025;</a> <a href="https://academic.oup.com/ve/article/9/2/vead064/7334491">Virus Evolution 2023)</a>.
							</p>	
						<p style="text-indent: 2em; margin-bottom: 15px;color: #000;"> The evolutionary history of giant viruses remains a subject of ongoing debate and uncertainty. The hybrid hypothesis for ICTV system suggests that giant viruses evolved from smaller eukaryotic dsDNA viruses.
						<p style="text-indent: 2em; margin-bottom: 15px;color: #000;"> The mosaic feature of mirusviruses suggests that herpesviruses evolved from tailed bacterial viruses via ancestral protist-infecting viruses, namely the ancestors of the modern day mirusviruses and herpesviruses <a href="https://www.nature.com/articles/s41586-023-05962-4">(Nature 2023)</a>. Then we detected mirusviruses in a broad range of eukaryotic genome assemblys <a href="https://pubmed.ncbi.nlm.nih.gov/38806056/">(Curr Biol 2024)</a>.
						<p/>
					</div>
				</section>

				<!-- Two -->
					<section id="two" style="display: flex; flex-wrap: wrap;">
						<!-- Profile Information Section -->
						<div class="profile-info" style="flex-basis: 60%; padding: 20px; font-size: 14pt;">
							<span style="font-weight: 1000; font-size: 18pt;">Spatiotemporal distribution</span>
							<br /><br />
							<p style="text-indent: 2em; margin-bottom: 15px;color: #000;"> Giant viruses are widespread, abundant, and active in the ocean. A considerable proportion of unique populations have been found specifically in the Arctic Ocean. In my Ph.D. thesis, I aimed to understand the mechanisms by which these viruses form distinct communities in polar environments <a href="https://www.nature.com/articles/s41467-023-41910-6">(Nat. Comm. 2023)</a>.
							Besides the open ocean, giant viruses have also been detected in various aquatic environments, such as inlet<a href="https://journals.asm.org/doi/10.1128/msystems.01168-24">(mSystems 2024)</a> and deep freshwater lakes <a href="https://academic.oup.com/ismej/advance-article/doi/10.1093/ismejo/wrae182/7766090">(ISME J 2024)</a>. Giant viruses exhibit specific dark water temporal patterns. Overall, aquatic ecosystems harbor far more giant viruses than terrestrial or host-associated environments. However, a pithovirus lineage was discovered in numerous clinical patient samples over a three-year period, and was simultaneously detected in widespread underground water samples (coming soon). 
							In addition, other eukaryotic viruses, such as PLVs (Polinton-like viruses), are also abundant in aquatic ecosystems.</p>
							<p style="text-indent: 2em; margin-bottom: 15px;color: #000;"> A coastal Imitervirales community exhibits synchronous seasonal cycles with eukaryotes and year-round recurrence; nonetheless, most individual viral populations tend to be specialists rather than generalists</p>
							<p style="text-indent: 2em; margin-bottom: 15px;color: #000;"> How are these viruses so successful in aquatic ecosystems?
							Bt far I can gvie two strategies: first, by largely shifting their gene repertoires to adapt to changing hosts and environments; 
							and second, by accumulating sufficient microdiversity, which enhances their resilience and adaptability within complex ecosystems. Do I believe they are all the answer? Definately no.</p>
							<p>	
						</div>

						<!-- Image Section -->
						<div class="profile-photo" style="flex-basis: 40%; display: flex; justify-content: center; align-items: center; flex-direction: column;">
							<img src="../images/research/GV_distribution.png" alt="Lingjie Meng" style="max-width: 100%; height: auto; border-radius: 10px;" />
							<header class="major" style="text-align: center; margin-top: 10px; font-size: 8pt;">
								<h1><span style="font-weight: 1000;">Distribution of marine giant viruses<br /></span></h1>
							</header>
						</div>
					</section>

				<!-- Three -->
					<section id="three" style="display: flex; flex-wrap: wrap;">
						<!-- Image Section -->
						<div class="profile-photo" style="flex-basis: 30%; display: flex; justify-content: center; align-items: center; flex-direction: column;">
							<img src="../images/research/host.png" alt="Lingjie Meng" style="max-width: 100%; height: auto; border-radius: 10px;" />
							<header class="major" style="text-align: center; margin-top: 10px; font-size: 8pt;">
								<h1><span style="font-weight: 1000;"> Host range of giant viruses (Sun et. al. 2020)<br />
							</header>
						</div>

						<!-- Profile Information Section -->
						<div class="profile-info" style="flex-basis: 70%; padding: 20px; font-size: 14pt;">
							<span style="font-weight: 1000; font-size: 18pt;">Virus-Host interaction</span>
							<br /><br />
							<p style="text-indent: 2em; margin-bottom: 15px;color: #000;">We are familiar with classic predator–prey dynamics in macroworld, such as pair between wolves and rabbits. But the microscopic world is equally fascinating. Viruses, unlike other cellular predators, can only replicate by infecting a host cell, making them entirely dependent on their hosts for survival and showing diversed living strategy with their hosts. </p>
							<p style="text-indent: 2em; margin-bottom: 15px; color: #000;"> One common viral strategy is to kill the host through lysis, a phenomenon often described by the "kill-the-winner" hypothesis. 
								This also applies to some giant viruses; for example, HaV has been shown to terminate blooms of <i>Heterosigma akashiwo</i> and demonstrated using metagenomics (Coming soon). 
								However, not all eukaryotic viruses follow this lytic path. Many viruses establish persistent infections or integrate into host genomes. </p> 
							<p style="text-indent: 2em; margin-bottom: 15px; color: #000;"> To uncover these intricate virus–host relationships, model virus-host systems are essential and this cannot be fully resolved through metagenomics alone. I am committed to contributing to this challenge by learning more and more techniques. </p> 
							<p style="text-indent: 2em; margin-bottom: 15px; color: #000;"> Although limited in experimental validation, bioinformatics provides valuable opportunities for predicting virus–host relationships. My first research on giant viruses focused on developing methods to evaluate and improve the accuracy of host predictions using co-occurrence networks and phylogenetic analysis <a href="https://journals.asm.org/doi/epub/10.1128/msphere.01298-20">(mSphere 2021)</a>. 
								In addition, approaches such as detecting HGT events <a href="https://www.nature.com/articles/s41586-023-05962-4">(Nature 2023)</a> and screening for endogenous viral elements also provide important clues for inferring virus–host associations <a href="https://pubmed.ncbi.nlm.nih.gov/38806056/">(Curr Biol 2024)</a> . </p>
					</section>

				<!-- Four -->
					<section id="four" style="display: flex; flex-wrap: wrap;">
						<!-- Profile Information Section -->
						<div class="profile-info" style="flex-basis: 70%; padding: 20px; font-size: 14pt;">
							<span style="font-weight: 1000; font-size: 18pt;">Viral functions</span>
							<br /><br />
							<p style="text-indent: 2em; margin-bottom: 15px;color: #000;"> Understanding viral functions is essential for revealing how viruses interact with their hosts, influence ecosystems and have evolved. 
							Most of people believe that all viruses need to do is replicate themselves. So why do giant viruses being gigantism in their genome sizes and functions, such as Pandoraviruses, encode more than 2,000 genes? This seems counterintuitive given their reliance on host machinery. 
							The presence of such large and complex genomes suggests there must be important trade-offs at related to host manipulation, autonomous functionality, or adaptation to diverse environments. 
							Understanding these trade-offs is key to uncovering the evolutionary strategies and ecological roles of giant viruses.	
							<p>	
						</div>

						<!-- Image Section -->
						<div class="profile-photo" style="flex-basis: 30%; display: flex; justify-content: center; align-items: center; flex-direction: column;">
							<img src="../images/research/fuctions.jpg" alt="Lingjie Meng" style="max-width: 100%; height: auto; border-radius: 10px;" />
							<header class="major" style="text-align: center; margin-top: 10px; font-size: 8pt;">
								<h1><span style="font-weight: 1000;">Metabolic enzymes and pathways in giant viruses (Belhaouari et. al. 2022)<br /></span></h1>
							</header>
						</div>
					</section>



		<!-- Footer -->
		<footer id="footer">
			<div class="inner">
				<ul class="icons">
					<li><a href="https://twitter.com/lingjie_marine" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
					<li><a href="https://github.com/LingjieEcoEvo" class="icon brands fa-github"><span class="label">Github</span></a></li>
					<li><a href="mailto:lingjie@kuicr.kyoto-u.ac.jp" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
				</ul>
				<ul class="copyright">
					<li>&copy; v202404</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
				</ul>
			</div>
		</footer>
		<!-- Scripts -->
			<script src="../assets/js/jquery.min.js"></script>
			<script src="../assets/js/jquery.poptrox.min.js"></script>
			<script src="../assets/js/browser.min.js"></script>
			<script src="../assets/js/breakpoints.min.js"></script>
			<script src="../assets/js/util.js"></script>
			<script src="../assets/js/main.js"></script>

	</body>
</html>